provisioner/ibm-s3fs-provisioner_test.go

```diff
@@ -103,17 +103,17 @@ const (
 	parameterStatCacheExpireSeconds = "ibm.io/stat-cache-expire-seconds"
 	parameterAutoCache              = "ibm.io/auto_cache"
 
-	optionChunkSizeMB             = "chunk-size-mb"
-	optionParallelCount           = "parallel-count"
-	optionMultiReqMax             = "multireq-max"
-	optionStatCacheSize           = "stat-cache-size"
-	optionS3FSFUSERetryCount      = "s3fs-fuse-retry-count"
-	optionTLSCipherSuite          = "tls-cipher-suite"
-	optionDebugLevel              = "debug-level"
-	optionCurlDebug               = "curl-debug"
-	optionKernelCache             = "kernel-cache"
-	optionOSEndpoint              = "object-store-endpoint"
-	optionRegion                  = "region"
+	optionChunkSizeMB        = "chunk-size-mb"
+	optionParallelCount      = "parallel-count"
+	optionMultiReqMax        = "multireq-max"
+	optionStatCacheSize      = "stat-cache-size"
+	optionS3FSFUSERetryCount = "s3fs-fuse-retry-count"
+	optionTLSCipherSuite     = "tls-cipher-suite"
+	optionDebugLevel         = "debug-level"
+	optionCurlDebug          = "curl-debug"
+	optionKernelCache        = "kernel-cache"
+	optionOSEndpoint         = "object-store-endpoint"
+	//optionRegion                  = "region"
```

Comment:
same here.. can we remove?

Reply:
kept it commented for reference if required